sig-docs/charter.md

```diff
@@ -0,0 +1,97 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io. Kubernetes documentation includes:
+  - Documentation of the core Kubernetes APIs
+  - Core Kubernetes architecture 
+  - CLI tools shipped with the Kubernetes release
+
+ Responsibility for creating feature documentation belongs to the developers and SIGs associated with a feature. This includes documentation for features, concepts, components, opinions, processes, etc.
+
+SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and [coordinates documentation updates][docs-release] during quarterly releases.
+
+SIG Docs maintains the Kubernetes website's infrastructure and tooling.
+
+SIG Docs creates subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
```

Comment:
This section is weirdly non-parallel, but I'm struggling to think of a good way to list it. Here's my swing at it: 

- The kubernetes.io website, which includes:
  - Site content and documentation
  - Content style guides, which are applied to feature documentation and release notes
  - Processes for launching feature content and release notes during quarterly releases
  - Site infrastructure (Hugo, Netlify) and tooling
  - Site analytics (Google Analytics)
  - Site styles and CSS 
  - Generated reference documentation for:
    .....